### PR TITLE
Testing my tests

### PR DESCRIPTION
**Pull request by brian-nguyen**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: go
+
+go:
+  - "1.9.2"
+
+script: go test ./...
```
Comment (bobheadxi) [inline .travis.yml]: Make sure you [install your dependencies too](https://github.com/ubclaunchpad/inertia/blob/master/.travis.yml#L33):
```
before_install:
  - go get -u github.com/mattn/goveralls
  - go get -u github.com/golang/dep/cmd/dep
  - make deps
```

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+language: go
+
+go:
+  - "1.9.2"
```
Comment (bobheadxi) [inline .travis.yml]: Unless there's a specific reason to use "1.9.x", might want to go with "1.10"

File: .travis.yml (hunk 1 above)
Comment (bobheadxi) [reply .travis.yml]: The `goveralls` dependency I pointed out is for [Coveralls coverage reports](https://github.com/ubclaunchpad/inertia/blob/master/.travis.yml#L42) btw, which is super nice to have:
```
script:
  - go test -race -coverprofile=coverage.out ./...  # Execute tests
  - goveralls -coverprofile=coverage.out -service=travis-ci -repotoken "$COVERALLS_TOKEN"
```

**Comment by brian-nguyen** [comment]
@bobheadxi Yes CI is always good 👍I'm just getting started and experimenting, so I appreciate your comments!!

**Comment by bobheadxi** [comment]
[Coverage Status](https://coveralls.io/github/ubclaunchpad/bumper?branch=brian%2Fci)

😎

**Pull request merged**